build.gradle

```diff
@@ -201,90 +197,75 @@ dependencies {
+    implementation 'com.azure:azure-storage-common:12.26.0'
+    implementation 'com.azure:azure-storage-file-datalake:12.20.0'
+    implementation 'com.azure:azure-data-tables:12.4.3'
 
     implementation platform('io.sentry:sentry-bom:7.13.0') //import bom
-    implementation('io.sentry:sentry-spring-boot-starter')
+    implementation('io.sentry:sentry-spring-boot-starter-jakarta')
     implementation('io.sentry:sentry-logback')
 
     // OpenTelemetry @WithSpan annotations:
-    implementation 'io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.2.0'
-
-    testImplementation 'org.apache.parquet:parquet-common:1.12.0'
-    testImplementation 'org.apache.parquet:parquet-hadoop:1.12.0'
-    testImplementation 'org.apache.parquet:parquet-hadoop-bundle:1.12.0'
-    testImplementation 'org.apache.parquet:parquet-encoding:1.12.0'
-    testImplementation 'org.apache.parquet:parquet-column:1.12.0'
-    testImplementation ('org.apache.hadoop:hadoop-common:3.3.1')  {
-        exclude group: 'com.sun.jersey', module: 'jersey-core'
-        exclude group: 'com.sun.jersey', module: 'jersey-servlet'
-        exclude group: 'com.sun.jersey', module: 'jersey-json'
-        exclude group: 'com.sun.jersey', module: 'jersey-server'
-    }
-    testImplementation ('org.apache.hadoop:hadoop-azure:3.3.1') {
+    implementation 'io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.6.0'
+
+    testImplementation 'org.apache.parquet:parquet-common:1.14.1'
+    testImplementation 'org.apache.parquet:parquet-hadoop:1.14.1'
+    testImplementation 'org.apache.parquet:parquet-hadoop-bundle:1.14.1'
+    testImplementation 'org.apache.parquet:parquet-encoding:1.14.1'
+    testImplementation 'org.apache.parquet:parquet-column:1.14.1'
+    testImplementation ('org.apache.hadoop:hadoop-common:3.4.0')  {
         exclude group: 'com.sun.jersey', module: 'jersey-core'
         exclude group: 'com.sun.jersey', module: 'jersey-servlet'
         exclude group: 'com.sun.jersey', module: 'jersey-json'
         exclude group: 'com.sun.jersey', module: 'jersey-server'
+        exclude group: 'org.slf4j', module: 'slf4j-reload4j'
     }
-    testImplementation('org.apache.hadoop:hadoop-mapreduce-client-core:3.3.1') {
-        exclude group: 'com.sun.jersey', module: 'jersey-core'
-        exclude group: 'com.sun.jersey', module: 'jersey-servlet'
-        exclude group: 'com.sun.jersey', module: 'jersey-json'
-        exclude group: 'com.sun.jersey', module: 'jersey-server'
-    }
-
-    testImplementation 'au.com.dius.pact.provider:junit5:4.3.19'
-    testImplementation 'au.com.dius.pact.provider:junit5spring:4.3.19'
 
-    antlr 'org.antlr:antlr4:4.8'
+    testImplementation 'au.com.dius.pact.provider:junit5:4.6.12'
+    testImplementation 'au.com.dius.pact.provider:junit5spring:4.6.12'
 
-    // Need groovy on the class path for the logback config. Could use XML and skip this dependency,
```

Comment:
We use XML now, and logback hasn't supported groovy since 2021: https://logback.qos.ch/news.html#1.2.9